understanding/22/target-size-minimum.html

```diff
@@ -72,35 +72,49 @@ <h3>Spacing versus size</h3>
       </figure>
-      
-        <figure id="pointer=target-stacked-links">
-            <img src="img/pointer-target-example7-stacked-links.png" alt="A  bulleted list of stacked links with sufficient and insufficient offset (see legend below)" />
+
+        <figure id="pointer-target-stacked-links">
+            <img src="img/pointer-target-example7-stacked-links.png" alt="A bulleted list of stacked links with sufficient and insufficient offset (see legend below)" />
           <figcaption>The two examples show lists of stacked links as used in navigation, for example in a list of contents of a longer text page. In both cases, the font size is 14px. In the first example, the line height is 25.8px, exceeding the requirement of 24 CSS pixels. In the second example, the stacked links have a line height of just 16.8px. They do not meet the vertical offset requirement of 24 CSS pixels.</figcaption>
       </figure>
 
         <p>Users with different disabilities have different needs for control sizes. It can be beneficial to provide an option to increase the active target area without increasing the visible target size. Another option is to provide a mechanism to control the density of layout and thereby change target size or spacing, or both. This can be beneficial for a wide range of users. For example, users with visual field loss may prefer a more condensed layout with smaller sized controls while users with other forms of low vision may prefer large controls.</p>
-        
-        
+
+        <p>The aim of this Success Criterion is that targets are ideally at least 24 by 24 CSS pixels. However, the offset exception allows for smaller targets, if there is sufficient spacing around them. Authors should be aware that using smaller targets can have an impact on layouts, as the combined space of the smaller target and the required spacing around it will generally require an overall area that is larger than 24 by 24 CSS pixels, the smaller the actual target is.</p>
+
+        <figure id="target-size-minimum">
+          <img src="img/target-size-minimum-diagram.png"
+            alt="A diagram showing target size of 4 targets, side-by-side that are 24 by 24 pixels in size. Another diagram below the first shows the same targets, reduced in size to 10 by 10 pixels, spaced apart by 14 pixels on all sides showing adjacent button spacing." />
+            <figcaption>
+              Five square targets with dimensions of 24 by 24 pixels, arranged in a cross (one target at the center, and one target above, one to the right, one below, and one to the left), without gaps - this passes. In the second diagram, the central target is shrunk to 10 by 10 pixels, with a margin of 7 pixels to the other targets - this will fail the Success Criterion.
```

Comment:
this `<figcaption>` is wrong though ... it describes the other image, not this one here?